v23.1/execution-locality-backups.md

```diff
@@ -0,0 +1,100 @@
+toc: true
+docs_area: manage
+---
+
+{% include_cached new-in.html version="v23.1" %} Using the `EXECUTION LOCALITY` option when you create a [backup](backup.html) job, you can set [locality filter](cockroach-start.html#locality) requirements that a node must meet in order to take part in executing a backup job. This will pin the [coordination of the backup job](backup-architecture.html#job-creation-phase) and the nodes that [process the row data](backup-architecture.html#export-phase) to the defined locality. 
+
+Defining an execution locality for a backup job, could be useful in the following cases:
+
+- When nodes in a cluster operate in different locality tiers, networking rules can restrict nodes from accessing a storage bucket. For an example, see [Access backup storage restricted by network rules](#access-backup-storage-restricted-by-network-rules).
+- When a multi-region cluster is running heavy workloads and an aggressive backup schedule, designating a region as the "backup" locality may improve latency. For an example, see [Create a non-primary region for backup jobs](#create-a-non-primary-region-for-backup-jobs).
+
+{{site.data.alerts.callout_info}}
+See [Take and Restore Locality-aware Backups](take-and-restore-locality-aware-backups.html) to run backup jobs where each node writes files only to the backup storage bucket that matches the node locality configured at node startup.
+{{site.data.alerts.end}}
+
+## Syntax
+
+To specify the locality requirements for the coordinating node, run `EXECUTION LOCALITY` with key-value pairs. The key-value pairs represent the [locality designations](cockroach-start.html#locality) assigned to the cluster at startup.
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+BACKUP DATABASE {database} INTO 'external://backup_storage' WITH EXECUTION LOCALITY = 'region={region},cloud={cloud}';
+~~~
+
+When you run `EXECUTION LOCALITY`, consider the following:
+
+- The backup job will fail if no nodes match the locality filter.
+- Selection of the coordinating node that matches the locality filter may slightly increase the startup latency of the backup job.
+- Even though a backup job has been pinned to a locality, it may still read data from another locality if there are no replicas in its locality.
+- If the job is created on a node that does not match the locality filter, you will receive an error even when the **job creation was successful**. This error indicates that the job execution moved to another node. To avoid viewing this error, we recommend using the [`DETACHED`](backup.html#detached) option with `EXECUTION LOCALITY` when creating a manual job rather than a [scheduled job](create-schedule-for-backup.html). Then, use the [`SHOW JOB WHEN COMPLETE`](show-jobs.html#show-job-when-complete) statement to determine when the job has finished. See [Job coordination using the execution locality option](#job-coordination-using-the-execution-locality-option) for more detail.
```

Comment:
```suggestion
- If the job is created on a node that does not match the locality filter, you will receive an error even when the **job creation was successful**. This error indicates that the job execution moved to another node. To avoid this error when creating a manual job (as opposed to a [scheduled job](create-schedule-for-backup.html)), you can use the [`DETACHED`](backup.html#detached) option with `EXECUTION LOCALITY`. Then, use the [`SHOW JOB WHEN COMPLETE`](show-jobs.html#show-job-when-complete) statement to determine when the job has finished. For more details, refer to [Job Coordination Using the `EXECUTION LOCALITY` option](#job-coordination-using-the-execution-locality-option).
```